app/views/users/show.html.erb

```diff
@@ -1,2 +1,8 @@
-<h1>The Best BlogPress User</h1>
-<p>Know More About Your Best Author Here</p>
+<div class="w-full flex flex-col items-center">
+<%=render "shared/user_info"%>
+ <%=render "shared/user_post"%>
+
+  <button class="bg-gray-300 text-white px-4 py-2 mb-4 rounded hover:text-gray-900" onclick="window.location.href='<%= user_posts_path(@user) %>'">
+    See all posts
+  </button>
+</div>
```

Comment:
- [ ] Let's ensure that the design for `user#show` page matches the given wireframe.

**Your design:**        |  **Expected:**
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/AlierPM/BlogPress/assets/12745474/5692ea7e-a82a-437b-9a6f-a3e448db8e20"> | <img width="800" alt="Home Page" src="https://github.com/AlierPM/BlogPress/assets/12745474/d61b1928-e830-4cd1-8349-11f96056ef9a">